### ISSUE TITLE
[MESH] Test Procedure in MESH/NODE/CFG/HBS/BV-02-C

### ISSUE THREAD

**Issue by MariuszSkamra** [intel/auto-pts#181]
CASE0040580 - Test Spec Errata

13th step says to repeat steps 1-12, so that in step 2 IUT is expected to send message with  the CountLog field is set to 0x00, but this is not valid.
It's because in the first iteration (executing 1-12 for the first time), IUT receives HB messages, so the count log is inctemented. Executing step 1 (as a part of 13th step) will leave CountLog unchanged as shall be (according to the MESH Profile Specification v1.0):
"If the Source or the Destination field is set to the unassigned address, or the PeriodLog field is set to 0x00, then the processing of received Heartbeat messages shall be disabled, the Heartbeat Subscription Source state shall be set to the unassigned address, the Heartbeat Subscription Destination state shall be set to the unassigned address, the Heartbeat Subscription MinHops state shall be unchanged, the Heartbeat Subscription MaxHops state shall be unchanged, and the Heartbeat Subscription Count state shall be unchanged."
To sum up CountLog shall not be 0x00 after 13th step, so 2th step is invalid then.

Affected TC  MESH/NODE/CFG/HBS/BV-02-C

**Comment by tedd-an** [comment]
Test pass and close it for now.
Reopen if you still see the issue.
